src/test/java/systems/composable/dropwizard/cassandra/DropwizardCassandraIntegrationTest.java

```diff
@@ -36,22 +37,51 @@
 
     @ClassRule
     public static final DropwizardAppRule<SmokeTestConfiguration> APP =
-            new DropwizardAppRule<>(SmokeTestApp.class, Resources.getResource("minimal.yml").getPath());
+        new DropwizardAppRule<>(SmokeTestApp.class, Resources.getResource("minimal.yml").getPath());
 
-    @Test
-    public void canQueryCassandra() throws Exception {
-        final URI uri = UriBuilder.fromUri("http://localhost")
-                .port(APP.getLocalPort())
-                .path("query")
-                .build();
+    @ClassRule
+    public static final DropwizardAppRule<SmokeTestConfiguration> APP_INJ =
```

Comment:
Ah, I see you've specified the ports in the `injected.yml` config file. I still think they belong in a separate test class though.